### PR TITLE
Add Appwrite's Hacktoberfest Swag 2019

### LINKED ISSUE #319: Hacktoberfest 2019: Appwrite
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : Appwrite

Q. Where have you seen the announcement (URL)?  
A. : https://medium.com/appwrite-io/hacktoberfest-2019-is-almost-here-lets-celebrate-it-together-24b311236dd

Q. What kind of swag (clothing, stickers, ...)?  
A. : stickers, badges

Q. Have you received swag from this?  
A. : no

Q. How long did it take?  
A. : N/A

Q. Does it ship worldwide?  
A. : Dunno

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by Yash-Garg**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Closes #319

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -350,6 +350,15 @@
     "dateAdded": "2019-10-04T16:13:00.000Z",
     "tags": ["hacktoberfest", "stickers", "accessories"]
   },
+  {
+    "name": "Appwrite",
```
Comment (vikaspotluri123) [inline data.json]: Alphabetical order please 😄

File: data.json [hunk 2]
```diff
@@ -350,6 +350,15 @@
...
+    "difficulty": "medium",
+    "description": "Make pull requests to issues tagged with \"Hacktoberfest\" or \"help wanted\" label in appwrite repo during October and get some stickers and badges.",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "description": "Make pull requests to issues tagged <code>Hacktoberfest<code> or <code>help wanted</code> in the <a href="https://github.com/appwrite/appwrite">appwrite repo</a> during October to get some stickers and badges.",
```

File: data.json (hunk 1 above)
Comment (Yash-Garg) [reply data.json]: Oh okay

File: data.json (hunk 2 above)
Comment (Yash-Garg) [reply data.json]: @vikaspotluri123 The hyperlink is making the build fail

Comment (vikaspotluri123) [reply data.json]: It's because I forgot to escape the quotes 🙃

Comment (Yash-Garg) [reply data.json]: AH xD

**Comment by Yash-Garg** [comment]
Squash and merge :")

File: data.json [hunk 3]
```diff
@@ -35,6 +35,15 @@
...
     "tags": ["stickers"]
   },
+  {
+    "name": "Appwrite",
+    "difficulty": "medium",
+    "description": "Make pull requests to issues tagged <code>Hacktoberfest</code> or <code>help wanted</code> in the <a href='https://github.com/appwrite/appwrite'>appwrite repo</a> during October to get some stickers and badges.",
+    "reference": "https://medium.com/appwrite-io/hacktoberfest-2019-is-almost-here-lets-celebrate-it-together-24b311236dd",
+    "image": "https://miro.medium.com/max/1600/1*mk5pBbuAZUg9j7nc6lbfrg.jpeg",
+    "dateAdded": "2019-10-06T20:30:00.000Z",
+    "tags": ["hacktoberfest", "stickers", "badges"]
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "tags": ["badges", "hacktoberfest", "stickers"]
```

File: data.json [hunk 4]
```diff
@@ -35,6 +35,15 @@
     "dateAdded": "2018-12-09T07:08:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "Appwrite",
+    "difficulty": "medium",
+    "description": "Make pull requests to issues tagged <code>Hacktoberfest</code> or <code>help wanted</code> in the <a href='https://github.com/appwrite/appwrite'>appwrite repo</a> during October to get some stickers and badges.",
```
Comment (aslafy-z) [inline data.json]:
```suggestion
    "description": "Make pull requests to issues tagged <code>hacktoberfest</code> or <code>help wanted</code> in the <a href='https://github.com/appwrite/appwrite'>appwrite repo</a> during October to get some stickers and badges.",
```

File: data.json (hunk 3 above)
Comment (vikaspotluri123) [reply data.json]: I was going to make all the changes at once later, but we can make them here too 👍

**Comment by aslafy-z** [comment]
Thanks @Yash-Garg! :100:

**Pull request merged**